enhancements/authentication/login-logout-events.md

```diff
@@ -42,67 +43,21 @@ This approach has a couple of drawbacks:
 
 ### Goals
 
-1. Add capabilities in oauth-server to create audit-like logs. Gate fine grained audit log events via a simple profile configuration mechanism.
+1. Add capabilities in oauth-server to create audit-like logs. Reuse audit profile configuration mechanism.
```

Comment:
> Reuse audit profile configuration mechanism.

Reuse which audit profile configuration mechanism? I think this is an implementation detail, should not appear in goals.